tensorflow/lite/micro/tools/ci_build/test_cortex_m_corstone_300.sh

```diff
@@ -51,4 +51,5 @@ readable_run make -j$(nproc) -f tensorflow/lite/micro/tools/make/Makefile \
   TARGET_ARCH=${TARGET_ARCH} \
   TOOLCHAIN=${TOOLCHAIN} \
   GENERIC_BENCHMARK_MODEL_PATH=tensorflow/lite/micro/models/person_detect.tflite \
+  GENERIC_BENCHMARK_ARENA_SIZE=`expr 150 \* 1024` \
```

Comment:
Good catch with `GENERIC_BENCHMARK_ARENA_SIZE`.  I failed to test locally without it, and it turns out the simulator just hangs (arena size is larger than mapped address space?).

Reply:
yes it allocated 3e6 which is way bigger than memory in the platform linker script